client/src/components/Profiles.js

```diff
@@ -58,6 +59,18 @@ const Profiles = () => {
                 ></Table.Cell>
                 <Table.Cell children={profile.phone_number}></Table.Cell>
                 <Table.Cell children={profile.email}></Table.Cell>
+                <Table.Cell>
+                  {" "}
+                  <Button
+                    as={Link}
+                    to={`/profiles/${profile.id}`}
+                    size="mini"
+                    basic
+                    color="black"
+                  >
+                    View
+                  </Button>{" "}
```

Comment:
```suggestion
                  </Button>
```
Let's remove the space as it's not doing anything